### ISSUE TITLE
Accessibility:: Unable to focus on Correct Answer button 

### ISSUE THREAD

**Issue by amruta-thakur** [adaptlearning/adapt_framework#2205]
### Issue: 
Unable to focus on Correct Answer button upon submission.

### Your environment
* adapt_framework: 3.2.0 
* All

### Steps to reproduce
- Turn on the accessibility mode.
- Make a setting of question component as "_canShowModelAnswer": true and "_canShowFeedback": true.
- After submitting incorrect answer, the Correct Answer button appears but unable to focus on it with "Tab" or "Shift+Tab".

### Expected behaviour
The focus should go to the Correct Answer button as this is the ideal functionality.

### Actual behaviour
Unable to focus on Correct Answer button.

I have also found the same issue in the latest adapt_framework-accessibilityv4 package.

**Comment by oliverfoster** [comment]
Duplicate of https://github.com/adaptlearning/adapt_framework/issues/911
